lib/PuppeteerSharp.Tests/BrowserContextTests/DefaultBrowserContextTests.cs

```diff
@@ -29,7 +29,7 @@ public async Task PageGetCookiesAsyncShouldWork()
             await Page.GoToAsync(TestConstants.EmptyPage);
 
             await Page.EvaluateExpressionAsync("document.cookie = 'username=John Doe'");
-            var cookie = (await Page.GetCookiesAsync()).FirstOrDefault();
+            var cookie = (await Page.GetCookiesAsync()).First();
```

Comment:
no null check after a `FirstOrDefault` implies it can never be empty, and hence should be a `First`